.github/workflows/docker_build_test_publish.yml

```diff
@@ -47,7 +47,7 @@ jobs:
       with:
         username: ${{ secrets.DOCKERHUB_USERNAME }}
         password: "${{ secrets.DOCKERHUB_PASSWORD }}"  # you don't need to manually set this secret. GitHub does it on your behalf
-        image_name: swedbankpay/jekyll-plantuml
+        image_name: jekyll-plantuml
         image_tag: ${{ steps.version.outputs.version }}
         dockerfile: ".docker/Dockerfile"
         push_image_and_stages: docker run --tty --volume ${{ github.workspace }}/test/jekyll-plantuml:/srv/jekyll swedbankpay/jekyll-plantuml:${{ steps.version.outputs.version }} jekyll build
```

Comment:
The hostname is optional here because we're pushing to `docker.com`?

Reply:
Yup, that's the default. I can add it to be explicit if wanted.

Reply:
Nah, I think it's okay. As long as it works. 😄

Reply:
The last deploy worked at least 🙏